hs_core/hydroshare/resource.py

```diff
@@ -770,6 +774,18 @@ def update_science_metadata(pk, metadata):
     resource.metadata.update(metadata)
 
 
+# delete from solr index
```

Comment:
This function is obsolete and needs to be removed. It was a result of a prior attempt at real-time indexing that is no longer needed.